baklava/package.json

```diff
@@ -59,6 +59,7 @@
     }
   },
   "dependencies": {
+    "@trodi/electron-splashscreen": "^1.0.0",
```

Comment:
I'm not seeing this used anywhere.
Unless I'm mistaken, it is an unused module and can be removed?